Cargo.toml

```diff
@@ -30,11 +30,14 @@ repository = "https://github.com/apache/skywalking-rust"
 async-stream = "0.3.3"
 base64 = "0.13.0"
 bytes = "1.1.0"
+futures-core = "0.3.21"
+futures-util = "0.3.21"
 prost = "0.10.4"
 prost-derive = "0.10.1"
 thiserror = "1.0.31"
 tokio = { version = "1.18.2", features = ["full"] }
-tonic = "0.7.2"
+tonic = { version = "0.7.2", features = ["codegen"] }
+tracing = "0.1.35"
```

Comment:
Why aren't these new dependencies detected by skywalking eyes? I think these are new dependencies, no?

Reply:
These dependencies actually already exist (required by the dependencies explicitly declared in `Cargo.toml` ), I just re-import the crate name to use, run `Cargo tree` can see the dependencies tree.